shell/src/main/java/org/apache/accumulo/shell/Shell.java

```diff
@@ -242,9 +242,8 @@
     }
   }
 
-  public Shell() throws IOException {
-    this(new ConsoleReader());
-  }
+  // no arg constructor should do minimal work since its used in Main ServiceLoader
+  public Shell() throws IOException {}
```

Comment:
Can remove the `throws IOException` as it won't affect callers at all. They were already expecting the IOException -- if we don't throw that anymore, that's fine.